ampl-converter/src/main/java/com/powsybl/ampl/converter/AmplNetworkReader.java

```diff
@@ -41,15 +38,17 @@
     private final DataSource dataSource;
 
     private final Network network;
+    private int variantIndex;
 
     private final StringToIntMapper<AmplSubset> mapper;
     private final Map<String, Bus> buses;
 
-    public AmplNetworkReader(DataSource dataSource, Network network, StringToIntMapper<AmplSubset> mapper) {
+    public AmplNetworkReader(DataSource dataSource, Network network, int variantIndex, StringToIntMapper<AmplSubset> mapper) {
```

Comment:
Create a new constructor that calls the new one, using the constant we've defined in AmplConstants like it's done in `AmplNetworkWriter` 
Thus, you can revert all the changes you made in the unit test